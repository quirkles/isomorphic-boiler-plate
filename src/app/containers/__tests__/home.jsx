import React from 'react';
import { mount } from 'enzyme';

import Home from '../Home';

describe('Home component', () => {
  it('Renders without errors and matches snapshot', () => {
    let wrapper;
    expect(() => {
      wrapper = mount(<Home />);
    }).not.toThrow();
    expect(wrapper).toMatchSnapshot();
  });
});
