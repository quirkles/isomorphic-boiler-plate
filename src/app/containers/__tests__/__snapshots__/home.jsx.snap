// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component Renders without errors and matches snapshot 1`] = `
<Home>
  <div>
    Hello world
  </div>
</Home>
`;
